PlayCover/Utils/IPA.swift

```diff
@@ -87,21 +118,29 @@ public class IPA {
             let stringArray = appLookup.components(separatedBy: CharacterSet.decimalDigits.inverted)
             appID = Int(stringArray.last ?? "0") ?? 0
         }
+        let supportMacOS: Bool = await checkMacOSCompatibility(appID: appID)
         let noMacAlert = UserDefaults.standard.bool(forKey: "\(bundleID).noMacAlert")
-        if PlayApp.MACOS_APPS.contains(bundleID), !noMacAlert {
+        let showAlert = InstallPreferences.shared.showAppStorePopup
+        if showAlert && supportMacOS && !noMacAlert {
             let alert = NSAlert()
-            alert.messageText = NSLocalizedString("alert.error", comment: "")
```

Comment:
The alert message text should be something along the lines of `MacOS version available`.

Reply:
<img width="404" alt="image" src="https://github.com/user-attachments/assets/9bee067f-7f30-4d84-ae46-c54ca38c0310" />

Like this?